frontend/src/lib/lemon-ui/Link/Link.tsx

```diff
@@ -21,6 +21,8 @@ export type LinkProps = Pick<React.HTMLProps<HTMLAnchorElement>, 'target' | 'cla
     to?: string | [string, RoutePart?, RoutePart?]
     /** If true, in-app navigation will not be used and the link will navigate with a page load */
     disableClientSideRouting?: boolean
+    /** If true, docs links will not be opened in the docs panel */
+    disableDocsPanel?: boolean
```

Comment:
This is part of an exception to allow links generated by Max to open in a new tab, since there's code to force links from the `Docs` panel to ignore `target="_blank`.  This is part of the exception carved out for links that Max includes in his answers so they'll open in a new tab (currently we'd lose the chat session if the links opened in the sidepanel instead of a new tab.)

The key difference:
- `disableDocsPanel` is specific to documentation links and controls whether they open in the side panel
- `disableClientSideRouting` is a broader setting that affects all navigation, determining whether links should trigger a full page load or use client-side routing